src/nft/components/profile/list/MarketplaceRow.tsx

```diff
@@ -136,13 +136,16 @@ export const MarketplaceRow = ({
   toggleExpandMarketplaceRows,
   rowHovered,
 }: MarketplaceRowProps) => {
-  const [listPrice, setListPrice] = useState<number>()
-  const [globalOverride, setGlobalOverride] = useState(false)
-  const showGlobalPrice = globalPriceMethod === SetPriceMethod.SAME_PRICE && !globalOverride && globalPrice
   const setAssetListPrice = useSellAsset((state) => state.setAssetListPrice)
   const removeAssetMarketplace = useSellAsset((state) => state.removeAssetMarketplace)
   const [marketIconHovered, toggleMarketIconHovered] = useReducer((s) => !s, false)
   const [marketRowHovered, toggleMarketRowHovered] = useReducer((s) => !s, false)
+  const existingPrice = asset.newListings?.find((listing) =>
+    expandMarketplaceRows ? listing.marketplace.name === selectedMarkets?.[0].name : !!listing.price
+  )?.price
+  const [listPrice, setListPrice] = useState<number | undefined>(existingPrice)
```

Comment:
since the find function is going to run on existing renders we can optimize this slightly by doing this: 

`const [listPrice, setListPrice] = useState<number | undefined>(() => asset.newListings?.find((listing) =>
    expandMarketplaceRows ? listing.marketplace.name === selectedMarkets?.[0].name : !!listing.price
  )?.price)`
  
  now it will only run on initial render